hedera-mirror-rest/monitoring/common.js

```diff
@@ -85,10 +85,8 @@ const filterTestDetails = (result, resultType = TEST_RESULT_TYPES.FAILED) => {
  */
 const getStatus = (resultType) => {
   const results = Object.values(currentResults).map((result) => filterTestDetails(result, resultType));
-  const httpErrorCodes = results
-    .map((result) => result.httpCode)
-    .filter((httpCode) => httpCode < 200 || httpCode > 299);
-  const httpCode = httpErrorCodes.length === 0 ? 200 : 409;
+  const successfulTests = results.map((result) => result.results.success).filter((success) => success === true);
+  const httpCode = successfulTests.length === results.length ? 200 : 409;
```

Comment:
Can change to fail fast. Something like:
```suggestion
  const failed = Objects.values(currentResults).find((r) => !r.results.success);
  const httpCode = failed ? 409 : 200;
```

Reply:
This is addressed. 
When I do run` /api/v1/status` I do this in the logs
` GET /api/v1/status/local returned 409` but the curl or the browser does not return the httpCode field for  `/api/v1/status` , but it does for `/api/v1/status/{name}`